src/Test.UnitTests.Sarif/GitHelperTests.cs

```diff
@@ -178,7 +178,7 @@ public void GitExePath_WhenPathDoesNotExist_SettingManuallyShouldWork()
         [Fact]
         public void GetTopLevel_WhenRepoPathIsToAFile()
         {
-            string pathToFile = typeof(GitHelperTests).Assembly.Location;
+            string pathToFile = Environment.CurrentDirectory;
```

Comment:
>Environment.CurrentDirectory [](http://example.com/codeflow?start=32&length=28)

will `Environment.CurrentDirectory` be the same as Assembly location? I think that depends on how it is executed, if the test run local and on server both work then I think it is good.